src/test/regress/expected/isolation_citus_dist_activity.out

```diff
@@ -45,16 +45,16 @@ step s3-view-worker:
 
 query          query_hostname query_hostport master_query_host_namemaster_query_host_portstate          wait_event_typewait_event     usename        datname        
 
-SELECT worker_apply_shard_ddl_command (105941, 'public', '
+SELECT worker_apply_shard_ddl_command (105949, 'public', '
```

Comment:
We might be missing a `SET citus.next_shard_id`

Reply:
sounds plausible, I will dive a bit deeper and use an unused range for these tests to make them more deterministic in their execution.

Reply:
Setting the shard id causes all bit the first permutation to fail due to the following messages in the output:
```
ERROR:  cached metadata for shard 107000 is inconsistent
```

I left a comment in the test file for the next soul that will run into this.